crates/subspace-runtime/src/lib.rs

```diff
@@ -726,6 +726,44 @@ impl SignedExtension for CheckStorageAccess {
     }
 }
 
+/// Controls non-root access to feeds and object store
+#[derive(Debug, Encode, Decode, Clone, Eq, PartialEq, Default, TypeInfo)]
+pub struct DisablePallets;
```

Comment:
The comment above is outdated. And `CheckCall`? Take it or leave it, not crucial.

Reply:
I thought about that and decided to make it more specific. Just like there is `pallet_transaction_payment::ChargeTransactionPayment`.